seshdash/views.py

```diff
@@ -806,3 +806,44 @@ def graphs(request):
         return HttpResponse(json.dumps(results))
     else:
         return HttpResponseBadRequest()
+
+#function to editing existing sites
+@login_required
+def edit_site(request,site_Id):
+    #creating an instance to populate a form
+    instance = get_object_or_404(Sesh_Site, id=site_Id)
+   
+    form = SiteForm(instance=instance)    
+    #checking if the form is valid
+   
+    if form.is_valid():
+        form = form.save()
+    return render(request,'seshdash/settings.html', {'form_edit':form})
+
+# function of adding new site
+@login_required
+def add_site(request):
+    #fetching list of sites for the user
+    user_sites = {}
+    user_site_name = []
+    user_site_id = []
+    sites =  _get_user_sites(request)
+    for site in sites:
+        user_site_name.append(site.site_name)
+        user_site_id.append(site.id)
+    user_sites = dict(zip(user_site_id,user_site_name))
+    # on ajax 
+    if request.method == 'POST':
+    
```

Comment:
Please get rid of unnecessary white spaces. Use whitespaces to break up the code into logical sections.